gravitee-apim-portal-webui-next/src/app/applications/application/application-tab-logs/application-log/application-log.component.html

```diff
@@ -21,97 +21,29 @@
-            </mat-expansion-panel-header>
-            @if (log.request?.body) {
-              <app-copy-code class="m3-body-medium" [text]="log.request?.body ?? ''" />
-            } @else {
-              <div class="m3-body-medium" i18n="@@applicationLogNoBody">No content logged.</div>
-            }
-          </mat-expansion-panel>
-        </mat-card-content>
-      </mat-card>
-      <mat-card appearance="outlined" class="application-log__container__column">
-        <mat-card-content class="application-log__container__column__content">
-          <div class="m3-title-medium" i18n="@@applicationLogResponse">Response</div>
-          <div class="m3-body-medium application-log__container__column__content__panel">
-            <div><span class="m3-title-small" i18n="@@applicationLogTransactionId">Transaction ID</span>: {{ log.id }}</div>
-            <div><span class="m3-title-small" i18n="@@applicationLogStatus">Status</span>: {{ log.status }}</div>
-            <div>
-              <span class="m3-title-small" i18n="@@applicationLogResponseTime">Response Time</span>: {{ log.responseTime
-              }}<span i18n="@@applicationLogMillisecond">ms</span>
-            </div>
-            <div>
-              <span class="m3-title-small" i18n="@@applicationLogContentLength">Content Length</span>: {{ log.responseContentLength }}
-            </div>
-          </div>
-          <mat-expansion-panel [expanded]="false" class="mat-elevation-z0" aria-label="Response Headers">
-            <mat-expansion-panel-header>
-              <mat-panel-title i18n="@@applicationLogHeaders">Headers</mat-panel-title>
-            </mat-expansion-panel-header>
-            <div class="m3-body-medium application-log__container__column__content__panel">
-              @for (header of log.responseHeaders; track header) {
-                <div>
-                  <span class="m3-title-small">{{ header.key }}: </span>{{ header.value }}
-                </div>
-              } @empty {
-                <span i18n="@@applicationLogNoHeaders">No headers logged.</span>
-              }
-            </div>
-          </mat-expansion-panel>
-          <mat-expansion-panel [expanded]="false" class="mat-elevation-z0" aria-label="Response Body">
-            <mat-expansion-panel-header>
-              <mat-panel-title i18n="@@applicationLogBody">Body</mat-panel-title>
-            </mat-expansion-panel-header>
-            @if (log.response?.body) {
-              <app-copy-code class="m3-body-medium" [text]="log.response?.body ?? ''" />
-            } @else {
-              <div class="m3-body-medium" i18n="@@applicationLogNoBody">No content logged.</div>
-            }
-          </mat-expansion-panel>
-        </mat-card-content>
-      </mat-card>
+    <div class="m3-body-medium application-log__row-one">
+      <div>
+        <span class="m3-title-small" i18n="@@applicationLogTimestamp">Timestamp</span>:
+        {{ log.timestamp | date: 'YYYY-MM-dd HH:mm:ss.SSS' }}
```

Comment:
I we succeed to define the format globally we can remove this format otherwise you can remove this comment 😅